ProofWidgets/Data/Html.lean

```diff
@@ -73,36 +75,80 @@ scoped syntax "<" ident jsxAttr* "/>" : jsxElement
 scoped syntax "<" ident jsxAttr* ">" jsxChild* "</" ident ">" : jsxElement
 
 scoped syntax jsxText      : jsxChild
--- TODO(WN): expand `{... $t}` as list of children
+/-- Interpolates an array of elements into a JSX literal -/
+scoped syntax "{..." term "}" : jsxChild
 /-- Interpolates an expression into a JSX literal -/
 scoped syntax "{" term "}" : jsxChild
 scoped syntax jsxElement   : jsxChild
 
 scoped syntax:max jsxElement : term
 
-def transformTag (n m : Ident) (ns : Array Ident) (vs : Array (TSyntax `jsxAttrVal))
+/-- Sends `#[a, b, c]` to `` `(term| $a ++ $b ++ $c)``-/
+def joinArrays {m} [Monad m] [MonadRef m] [MonadQuotation m] (arr : Array Term) : m Term := do
```

Comment:
Could we move the general functions such as `joinArrays`/`foldInlsM`/`delabListLiteral`/... to a `ProofWidgets.Util` module?

Reply:
I've moved the new ones in this PR.